executor/runtime/docker/docker_linux.go

```diff
@@ -154,3 +159,35 @@ func cleanupCgroups(cgroupPath string) error {
 
 	return nil
 }
+
+func setupContainerNesting(parentCtx context.Context, c *runtimeTypes.Container, cred ucred) error {
+	if !c.TitusInfo.GetAllowNestedContainers() {
+		return nil
+	}
+	cgroupPath := filepath.Join("/proc/", strconv.FormatInt(int64(cred.pid), 10), "cgroup")
+	cgroups, err := ioutil.ReadFile(cgroupPath)
+	if err != nil {
+		return err
+	}
+	var ret error
+	for _, line := range strings.Split(string(cgroups), "\n") {
+		cgroupInfo := strings.Split(strings.TrimSpace(line), ":")
+		if len(cgroupInfo) != 3 {
+			continue
+		}
+		controllerType := cgroupInfo[1]
+		if len(controllerType) == 0 {
+			continue
```

Comment:
log a `warn`?

Reply:
If you have lots of controller-less cgroups, you'll get a lot of not-useful warnings.